gnome_settings_backup.txt
[ca/desrt/dconf-editor]
saved-pathbar-path='/org/gnome/shell/extensions/wsmatrix/workspace-overview-toggle'
saved-view='/org/gnome/shell/extensions/wsmatrix/workspace-overview-toggle'
show-warning=false
window-height=1032
window-is-maximized=false
window-width=1920

[com/github/Ory0n/Resource_Monitor]
cpustatus=true
diskdeviceslist=['dev /dev false false', '/dev/nvme0n1p1 / false false', '/dev/nvme0n1p2 /home false false', '/dev/nvme0n1p3 /boot/efi false false']
diskspacestatus=false
diskstatsmode='multiple'
diskstatsstatus=false
extensionposition='right'
iconsposition='left'
iconsstatus=true
leftclickstatus='gnome-system-monitor'
netautohidestatus=true
rightclickstatus=true
swapstatus=false
thermalcputemperaturedeviceslist=['acpitz: temp1-false-/sys/class/hwmon/hwmon0/temp1_input', 'acpitz: temp2-false-/sys/class/hwmon/hwmon0/temp2_input', 'nvme: Composite-false-/sys/class/hwmon/hwmon1/temp1_input', 'iwlwifi_1: temp1-false-/sys/class/hwmon/hwmon2/temp1_input', 'coretemp: Package id 0-false-/sys/class/hwmon/hwmon3/temp1_input', 'coretemp: Core 0-false-/sys/class/hwmon/hwmon3/temp2_input', 'coretemp: Core 1-false-/sys/class/hwmon/hwmon3/temp3_input', 'coretemp: Core 2-false-/sys/class/hwmon/hwmon3/temp4_input', 'coretemp: Core 3-false-/sys/class/hwmon/hwmon3/temp5_input', 'coretemp: Core 4-false-/sys/class/hwmon/hwmon3/temp6_input', 'coretemp: Core 5-false-/sys/class/hwmon/hwmon3/temp7_input']
thermalcputemperaturestatus=false

[com/github/amezin/ddterm]
background-color='rgb(63,63,63)'
bold-is-bright=false
custom-font='Sarasa Fixed SC 12'
ddterm-toggle-hotkey=['<Super>F1']
foreground-color='rgb(220,220,204)'
hide-window-on-esc=false
palette=['rgb(63,63,63)', 'rgb(204,147,147)', 'rgb(127,159,127)', 'rgb(240,223,175)', 'rgb(140,208,211)', 'rgb(220,140,195)', 'rgb(147,224,227)', 'rgb(220,220,204)', 'rgb(127,127,127)', 'rgb(156,99,99)', 'rgb(95,127,95)', 'rgb(208,191,143)', 'rgb(148,191,243)', 'rgb(203,123,178)', 'rgb(130,207,210)', 'rgb(255,255,239)']
panel-icon-type='menu-button'
theme-variant='dark'
use-system-font=false
use-theme-colors=false
window-maximize=false
window-size=0.59999999999999998

[com/github/maoschanz/DynamicWallpaperEditor]
display-mode='list'

[de/haeckerfelix/Fragments]
client-last-connection='00000000-0000-0000-0000-000000000000'
window-height=822
window-width=1022

[desktop/ibus/general]
preload-engines=['xkb:us::eng', 'libpinyin']
version='1.5.25'

[io/github/celluloid-player/celluloid]
settings-migrated=true

[io/github/celluloid-player/celluloid/window-state]
height=534
loop-playlist=false
maximized=false
playlist-width=250
show-playlist=false
volume=0.80000000000000004
width=954

[io/github/realmazharhussain/GdmSettings]
never-applied=false

[io/github/realmazharhussain/GdmSettings/appearance]
background-color='rgb(40,40,40)'
background-image='/home/daiwz/.config/awesome/themes/zenburn/disco.png'
background-type='Image'
cursor-theme='Paper'
icon-theme='Numix-Circle'
shell-theme='default'

[io/github/realmazharhussain/GdmSettings/fonts]
antialiasing='rgba'
font='方正屏显雅宋_GBK 12'
hinting='medium'
scaling-factor=1.0

[io/github/realmazharhussain/GdmSettings/misc]
disable-restart-buttons=false
disable-user-list=false
enable-logo=false
enable-welcome-message=false
logo=''
welcome-message=''

[io/github/realmazharhussain/GdmSettings/night-light]
night-light-enabled=false
night-light-end-hour=6
night-light-end-minute=0
night-light-schedule-automatic=true
night-light-start-hour=20
night-light-start-minute=0
night-light-temperature=uint32 2700

[io/github/realmazharhussain/GdmSettings/pointing/mouse]
inverse-scrolling=false
mouse-speed=0.0
pointer-acceleration='default'

[io/github/realmazharhussain/GdmSettings/pointing/touchpad]
disable-while-typing=true
natural-scrolling=true
tap-to-click=false
touchpad-speed=0.0
two-finger-scrolling=true

[io/github/realmazharhussain/GdmSettings/sound]
event-sounds=true
feedback-sounds=false
over-amplification=false
sound-theme='freedesktop'

[io/github/realmazharhussain/GdmSettings/tools]
top-bar-tweaks=false

[io/github/realmazharhussain/GdmSettings/top-bar]
change-top-bar-background-color=false
change-top-bar-text-color=false
disable-top-bar-arrows=false
disable-top-bar-rounded-corners=false
show-battery-percentage=false
show-seconds=false
show-weekday=true
time-format='24h'
top-bar-background-color='rgb(0,0,0)'
top-bar-text-color='rgb(250,250,250)'

[io/github/realmazharhussain/GdmSettings/window-state]
height=uint32 1080
last-visited-page='appearance'
paned-position=uint32 190
width=uint32 1920

[net/launchpad/caffeine]
show-notification=true

[org/blueman/general]
plugin-list=['AppIndicator']
show-statusbar=true
window-properties=[954, 1072, 962, 4]

[org/blueman/network]
nap-enable=false

[org/blueman/plugins/autoconnect]
services=[('/org/bluez/hci0/dev_CC_98_8B_94_A8_F5', '00000000-0000-0000-0000-000000000000')]

[org/blueman/plugins/powermanager]
auto-power-on=@mb true

[org/blueman/plugins/recentconns]
recent-connections=[{'adapter': '08:6A:C5:15:F6:75', 'address': 'CC:98:8B:94:A8:F5', 'alias': 'LE_WH-1000XM3', 'icon': 'audio-headset', 'name': '音频和输入配置文件', 'uuid': '00000000-0000-0000-0000-000000000000', 'time': '1654222709.9505136'}]

[org/gnome/Characters]
recent-characters=['✓', '❾', '❽', '❼', '❻', '❺', '❹', '❸', '❷', '❶', '🌒', '🌑']

[org/gnome/Chess]
height=672
last-played-as='white'
maximized=false
width=822

[org/gnome/Geary]
migrated-config=true
window-height=1080
window-maximize=false
window-width=1920

[org/gnome/baobab/ui]
is-maximized=false
window-size=(1920, 1080)

[org/gnome/boxes]
first-run=false
shared-folders="[<{'uuid': <'b62296d6-af31-4e37-b010-7102f02579e8'>, 'path': <'/home/daiwz/Downloads'>, 'name': <'Downloads'>}>, <{'uuid': <'b62296d6-af31-4e37-b010-7102f02579e8'>, 'path': <'/home/daiwz/Games'>, 'name': <'Games'>}>]"
view='icon-view'
window-maximized=false
window-position=[962, 4]
window-size=[954, 534]

[org/gnome/cheese]
burst-delay=1000
camera='C505e HD Webcam'
photo-x-resolution=1280
photo-y-resolution=960
selected-effect='凸出'
video-x-resolution=1280
video-y-resolution=960

[org/gnome/control-center]
last-panel='wifi'
window-state=(1920, 1080, false)

[org/gnome/desktop/a11y]
always-show-universal-access-status=false

[org/gnome/desktop/a11y/applications]
screen-reader-enabled=false

[org/gnome/desktop/a11y/magnifier]
mouse-tracking='proportional'

[org/gnome/desktop/app-folders]
folder-children=['Utilities', 'YaST']

[org/gnome/desktop/app-folders/folders/Utilities]
apps=['gnome-abrt.desktop', 'gnome-system-log.desktop', 'nm-connection-editor.desktop', 'org.gnome.baobab.desktop', 'org.gnome.Connections.desktop', 'org.gnome.DejaDup.desktop', 'org.gnome.Dictionary.desktop', 'org.gnome.DiskUtility.desktop', 'org.gnome.eog.desktop', 'org.gnome.Evince.desktop', 'org.gnome.FileRoller.desktop', 'org.gnome.fonts.desktop', 'org.gnome.seahorse.Application.desktop', 'org.gnome.tweaks.desktop', 'org.gnome.Usage.desktop', 'vinagre.desktop']
categories=['X-GNOME-Utilities']
name='X-GNOME-Utilities.directory'
translate=true

[org/gnome/desktop/app-folders/folders/YaST]
categories=['X-SuSE-YaST']
name='suse-yast.directory'
translate=true

[org/gnome/desktop/app-folders/folders/accessories]
apps=['yad-icon-browser.desktop', 'de.haeckerfelix.Fragments.desktop']
categories=['Utility']
excluded-apps=['gnome-layouts.desktop', 'manjaro-documentation.desktop', 'org.gnome.Nautilus.desktop', 'org.gnome.Maps.desktop']
name='Accessories'
translate=true

[org/gnome/desktop/app-folders/folders/games]
categories=['Game']
name='Games'
translate=true

[org/gnome/desktop/app-folders/folders/office]
categories=['Office']
name='Office'
translate=true

[org/gnome/desktop/app-folders/folders/system-tools]
apps=['qvidcap.desktop', 'qv4l2.desktop', 'bssh.desktop', 'bvnc.desktop', 'hp-uiscan.desktop']
categories=['System', 'Settings']
excluded-apps=['org.manjaro.pamac.manager.desktop']
name='System Tools'
translate=true

[org/gnome/desktop/background]
color-shading-type='solid'
picture-options='zoom'
picture-uri='file:///home/daiwz/.local/share/backgrounds/2022-05-17-00-19-14-DD.jpg'
picture-uri-dark='file:///home/daiwz/.local/share/backgrounds/2022-05-17-00-19-14-DD.jpg'
primary-color='#000000000000'
secondary-color='#000000000000'

[org/gnome/desktop/calendar]
show-weekdate=false

[org/gnome/desktop/input-sources]
mru-sources=[('ibus', 'libpinyin'), ('xkb', 'us')]
per-window=false
sources=[('xkb', 'us')]
xkb-options=@as []

[org/gnome/desktop/interface]
clock-show-seconds=false
clock-show-weekday=true
color-scheme='prefer-dark'
cursor-size=24
cursor-theme='Paper'
document-font-name='方正屏显雅宋_GBK 12'
enable-animations=true
enable-hot-corners=false
font-antialiasing='rgba'
font-hinting='medium'
font-name='方正屏显雅宋_GBK 12'
gtk-key-theme='Default'
gtk-theme='Equilux'
icon-theme='Numix-Circle'
locate-pointer=false
monospace-font-name='SauceCodePro Nerd Font Mono 12'
text-scaling-factor=1.0
toolbar-style='text'

[org/gnome/desktop/media-handling]
autorun-never=true
autorun-x-content-ignore=['x-content/audio-cdda', 'x-content/video-dvd', 'x-content/audio-player', 'x-content/image-dcf']
autorun-x-content-open-folder=@as []
autorun-x-content-start-app=['x-content/unix-software', 'x-content/ostree-repository']

[org/gnome/desktop/notifications]
application-children=['org-manjaro-pamac-manager', 'caffeine', 'google-chrome', 'baidunetdisk', 'xdg-desktop-portal-gtk', 'org-gnome-boxes', 'de-haeckerfelix-fragments', 'alacritty', 'teams', 'org-gnome-geary', 'com-qq-weixin-deepin', 'emacs', 'org-gnome-nautilus', 'org-gnome-chromegnomeshell', 'org-gnome-extensions-desktop', 'skypeforlinux', 'material-shell-papyelgringo', 'ca-desrt-dconf-editor', 'emacsclient', 'org-kde-okular', 'com-github-xournalpp-xournalpp', 'gnome-system-monitor', 'steam', 'org-gnome-fileroller', 'manjaro-settings-manager', 'pavucontrol', 'org-gnome-settings']
show-banners=false

[org/gnome/desktop/notifications/application/alacritty]
application-id='Alacritty.desktop'

[org/gnome/desktop/notifications/application/baidunetdisk]
application-id='baidunetdisk.desktop'

[org/gnome/desktop/notifications/application/blueman-manager]
enable=false

[org/gnome/desktop/notifications/application/ca-desrt-dconf-editor]
application-id='ca.desrt.dconf-editor.desktop'

[org/gnome/desktop/notifications/application/caffeine]
application-id='caffeine.desktop'

[org/gnome/desktop/notifications/application/com-github-xournalpp-xournalpp]
application-id='com.github.xournalpp.xournalpp.desktop'

[org/gnome/desktop/notifications/application/com-qq-weixin-deepin]
application-id='com.qq.weixin.deepin.desktop'

[org/gnome/desktop/notifications/application/de-haeckerfelix-fragments]
application-id='de.haeckerfelix.Fragments.desktop'

[org/gnome/desktop/notifications/application/emacs]
application-id='emacs.desktop'

[org/gnome/desktop/notifications/application/emacsclient]
application-id='emacsclient.desktop'

[org/gnome/desktop/notifications/application/gnome-network-panel]
application-id='gnome-network-panel.desktop'

[org/gnome/desktop/notifications/application/gnome-system-monitor]
application-id='gnome-system-monitor.desktop'

[org/gnome/desktop/notifications/application/google-chrome]
application-id='google-chrome.desktop'

[org/gnome/desktop/notifications/application/manjaro-settings-manager]
application-id='manjaro-settings-manager.desktop'

[org/gnome/desktop/notifications/application/material-shell-papyelgringo]
application-id='material-shell@papyelgringo.desktop'

[org/gnome/desktop/notifications/application/org-fontforge-fontforge]
application-id='org.fontforge.FontForge.desktop'

[org/gnome/desktop/notifications/application/org-gnome-boxes]
application-id='org.gnome.Boxes.desktop'

[org/gnome/desktop/notifications/application/org-gnome-chromegnomeshell]
application-id='org.gnome.ChromeGnomeShell.desktop'

[org/gnome/desktop/notifications/application/org-gnome-extensions-desktop]
application-id='org.gnome.Extensions.desktop.desktop'

[org/gnome/desktop/notifications/application/org-gnome-fileroller]
application-id='org.gnome.FileRoller.desktop'

[org/gnome/desktop/notifications/application/org-gnome-geary]
application-id='org.gnome.Geary.desktop'

[org/gnome/desktop/notifications/application/org-gnome-nautilus]
application-id='org.gnome.Nautilus.desktop'

[org/gnome/desktop/notifications/application/org-gnome-settings]
application-id='org.gnome.Settings.desktop'

[org/gnome/desktop/notifications/application/org-gnome-software]
enable=false

[org/gnome/desktop/notifications/application/org-gnome-tweaks]
application-id='org.gnome.tweaks.desktop'

[org/gnome/desktop/notifications/application/org-inkscape-inkscape]
application-id='org.inkscape.Inkscape.desktop'

[org/gnome/desktop/notifications/application/org-kde-okular]
application-id='org.kde.okular.desktop'

[org/gnome/desktop/notifications/application/org-manjaro-pamac-manager]
application-id='org.manjaro.pamac.manager.desktop'

[org/gnome/desktop/notifications/application/pavucontrol]
application-id='pavucontrol.desktop'

[org/gnome/desktop/notifications/application/skypeforlinux]
application-id='skypeforlinux.desktop'

[org/gnome/desktop/notifications/application/steam]
application-id='steam.desktop'

[org/gnome/desktop/notifications/application/teams]
application-id='teams.desktop'

[org/gnome/desktop/notifications/application/wemeetapp]
application-id='wemeetapp.desktop'

[org/gnome/desktop/notifications/application/xdg-desktop-portal-gtk]
application-id='xdg-desktop-portal-gtk.desktop'

[org/gnome/desktop/notifications/application/yelp]
enable=false

[org/gnome/desktop/notifications/application/zhumu]
application-id='Zhumu.desktop'

[org/gnome/desktop/peripherals/mouse]
accel-profile='default'

[org/gnome/desktop/peripherals/tablets/056a:037a]
output=['SGT', 'C FORCE', 'demoset-1']

[org/gnome/desktop/peripherals/touchpad]
two-finger-scrolling-enabled=true

[org/gnome/desktop/privacy]
disable-microphone=false
old-files-age=uint32 30
recent-files-max-age=-1

[org/gnome/desktop/screensaver]
color-shading-type='solid'
picture-options='zoom'
picture-uri='file:///home/daiwz/.local/share/backgrounds/2022-05-17-00-19-14-DD.jpg'
primary-color='#000000000000'
secondary-color='#000000000000'

[org/gnome/desktop/search-providers]
disabled=['org.gnome.Software.desktop']
sort-order=['org.gnome.Contacts.desktop', 'org.gnome.Documents.desktop', 'org.gnome.Nautilus.desktop']

[org/gnome/desktop/session]
idle-delay=uint32 0

[org/gnome/desktop/sound]
event-sounds=true
theme-name='__custom'

[org/gnome/desktop/wm/keybindings]
close=['<Shift><Super>q']
minimize=['<Super>minus']
move-to-monitor-down=@as []
move-to-monitor-left=@as []
move-to-monitor-right=@as []
move-to-workspace-1=['<Primary><Super>1']
move-to-workspace-10=['<Shift><Super>0']
move-to-workspace-2=['<Primary><Super>2']
move-to-workspace-3=['<Primary><Super>3']
move-to-workspace-4=['<Primary><Super>4']
move-to-workspace-5=['<Primary><Super>5']
move-to-workspace-6=['<Primary><Super>6']
move-to-workspace-7=['<Primary><Super>7']
move-to-workspace-8=['<Primary><Super>8']
move-to-workspace-9=['<Primary><Super>9']
move-to-workspace-down=@as []
move-to-workspace-last=@as []
move-to-workspace-left=['<Primary><Super>comma']
move-to-workspace-right=['<Primary><Super>period']
show-desktop=['<Super>z']
switch-applications=['<Alt>Tab']
switch-applications-backward=['<Shift><Alt>Tab']
switch-input-source=@as []
switch-input-source-backward=@as []
switch-panels=@as []
switch-panels-backward=@as []
switch-to-workspace-1=['<Super>1']
switch-to-workspace-2=['<Super>2']
switch-to-workspace-3=['<Super>3']
switch-to-workspace-4=['<Super>4']
switch-to-workspace-5=['<Super>5']
switch-to-workspace-6=['<Super>6']
switch-to-workspace-7=['<Super>7']
switch-to-workspace-8=['<Super>8']
switch-to-workspace-9=['<Super>9']
switch-to-workspace-left=['<Super>comma']
switch-to-workspace-right=['<Super>period']
switch-windows=['<Super>Tab']
switch-windows-backward=['<Shift><Super>Tab']
toggle-fullscreen=['<Super>equal']
toggle-maximized=['<Super>j']

[org/gnome/desktop/wm/preferences]
action-middle-click-titlebar='minimize'
auto-raise=true
button-layout='icon:minimize,maximize,close'
focus-mode='sloppy'
num-workspaces=9
titlebar-font='方正屏显雅宋_GBK 12'
workspace-names=['壹', '贰', '叁', '肆', '伍', '陆', '柒', '捌', '玖']

[org/gnome/evince/default]
window-ratio=(2.9871645274212368, 0.10942312712085617)

[org/gnome/evolution-data-server]
migrated=true
network-monitor-gio-name=''

[org/gnome/file-roller/dialogs/extract]
recreate-folders=true
skip-newer=false

[org/gnome/file-roller/listing]
list-mode='as-folder'
name-column-width=456
show-path=false
sort-method='name'
sort-type='ascending'

[org/gnome/file-roller/ui]
sidebar-width=200
window-height=1025
window-width=954

[org/gnome/gedit/plugins]
active-plugins=['sort', 'spell', 'filebrowser', 'openlinks', 'docinfo', 'modelines']

[org/gnome/gedit/plugins/filebrowser]
root='file:///'
tree-view=true
virtual-root="file:///home/daiwz/.local/share/Baldur's%20Gate%20II%20-%20Enhanced%20Edition"

[org/gnome/gedit/preferences/ui]
show-tabs-mode='auto'

[org/gnome/gedit/state/window]
bottom-panel-size=140
side-panel-active-page='GeditWindowDocumentsPanel'
side-panel-size=200
size=(954, 486)
state=87168

[org/gnome/gnome-system-monitor]
cpu-colors=[(uint32 0, '#e6194B'), (1, '#f58231'), (2, '#ffe119'), (3, '#bfef45'), (4, '#3cb44b'), (5, '#42d4f4'), (6, '#4363d8'), (7, '#911eb4'), (8, '#f032e6'), (9, '#fabebe'), (10, '#ffd8b1'), (11, '#fffac8'), (12, '#aaffc3'), (13, '#469990'), (14, '#000075'), (15, '#e6beff'), (16, '#a4fef3327999'), (17, '#f3327999c879'), (18, '#7999ebf5f332'), (19, '#f332d6f37999')]
current-tab='resources'
maximized=false
network-total-in-bits=false
show-dependencies=false
show-whose-processes='user'
window-state=(1920, 1080, 0, 0)

[org/gnome/gnome-system-monitor/disktreenew]
col-6-visible=true
col-6-width=0

[org/gnome/gnome-system-monitor/proctree]
columns-order=[0, 1, 2, 3, 4, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26]
sort-col=0
sort-order=0

[org/gnome/mutter]
center-new-windows=false
dynamic-workspaces=false
edge-tiling=false
overlay-key='Super_R'
workspaces-only-on-primary=false

[org/gnome/mutter/keybindings]
toggle-tiled-left=['<Super>Left']
toggle-tiled-right=['<Super>Right']

[org/gnome/mutter/wayland/keybindings]
restore-shortcuts=@as []

[org/gnome/nautilus/compression]
default-compression-format='zip'

[org/gnome/nautilus/preferences]
default-folder-viewer='list-view'
search-filter-time-type='last_modified'
search-view='list-view'

[org/gnome/nautilus/window-state]
initial-size=(1920, 1080)
maximized=false
sidebar-width=200

[org/gnome/nm-applet/eap/5e8b1938-c112-4e84-bc2f-f6b46597c524]
ignore-ca-cert=false
ignore-phase2-ca-cert=false

[org/gnome/settings-daemon/plugins/color]
night-light-enabled=false

[org/gnome/settings-daemon/plugins/media-keys]
area-screenshot=@as []
area-screenshot-clip=@as []
custom-keybindings=['/org/gnome/settings-daemon/plugins/media-keys/custom-keybindings/custom0/', '/org/gnome/settings-daemon/plugins/media-keys/custom-keybindings/custom1/', '/org/gnome/settings-daemon/plugins/media-keys/custom-keybindings/custom2/', '/org/gnome/settings-daemon/plugins/media-keys/custom-keybindings/custom3/', '/org/gnome/settings-daemon/plugins/media-keys/custom-keybindings/custom4/', '/org/gnome/settings-daemon/plugins/media-keys/custom-keybindings/custom5/', '/org/gnome/settings-daemon/plugins/media-keys/custom-keybindings/custom6/', '/org/gnome/settings-daemon/plugins/media-keys/custom-keybindings/custom7/', '/org/gnome/settings-daemon/plugins/media-keys/custom-keybindings/custom8/', '/org/gnome/settings-daemon/plugins/media-keys/custom-keybindings/custom9/']
help=@as []
magnifier=@as []
magnifier-zoom-in=@as []
magnifier-zoom-out=@as []
mic-mute=['Eject']
screenreader=@as []
screensaver=@as []
screenshot=@as []
screenshot-clip=@as []
volume-down=['AudioLowerVolume']
volume-mute=['AudioMute']
volume-up=['AudioRaiseVolume']
window-screenshot=@as []
window-screenshot-clip=@as []

[org/gnome/settings-daemon/plugins/media-keys/custom-keybindings/custom0]
binding='<Super>Return'
command='alacritty'
name='Alacritty'

[org/gnome/settings-daemon/plugins/media-keys/custom-keybindings/custom1]
binding='<Super>F2'
command='emacsclient -c -a emacs'
name='Emacs client'

[org/gnome/settings-daemon/plugins/media-keys/custom-keybindings/custom2]
binding='<Super>F3'
command='google-chrome-stable'
name='Chrome'

[org/gnome/settings-daemon/plugins/media-keys/custom-keybindings/custom3]
binding='<Super>F5'
command='nautilus'
name='Nautilus'

[org/gnome/settings-daemon/plugins/media-keys/custom-keybindings/custom4]
command='/home/daiwz/.scripts/myclip.sh'
name='Greenclip'

[org/gnome/settings-daemon/plugins/media-keys/custom-keybindings/custom5]
binding='Print'
command='flameshot full -c -p /home/daiwz/Pictures/Flameshot'
name='Print full screen'

[org/gnome/settings-daemon/plugins/media-keys/custom-keybindings/custom6]
binding='<Super>Print'
command='flameshot gui'
name='Print screen'

[org/gnome/settings-daemon/plugins/media-keys/custom-keybindings/custom7]
binding='<Primary><Super>c'
command='greenclip clear'
name='Clear clip board'


.spacemacs.d/hal9001/config.el
;;---------------------
;; Customize AucTex
;;---------------------
(setq TeX-auto-save t)
(setq TeX-parse-self t)
(setq-default TeX-master nil)
(setq TeX-source-correlate-mode t)

;; Enable synctex correlation
(setq TeX-source-correlate-method 'synctex)

(if (string-equal system-type "windows-nt")
    (require 'tex-mik))

;; hook mode
(add-hook 'LaTeX-mode-hook
          (lambda ()
            (setq TeX-auto-untabify t     ; remove all tabs before saving
                  TeX-engine 'default)       ; use xlatex default
                  ;TeX-show-compilation t) ; display compilation windows
            (TeX-PDF-mode t)       ; PDF mode enable, not plain
            (setq TeX-save-query nil)
            (imenu-add-menubar-index)
	          (setq TeX-auto-save t)
	          (setq TeX-parse-self t)
	          (setq-default TeX-master nil)
            (define-key LaTeX-mode-map (kbd "TAB") 'TeX-complete-symbol)))

(defadvice TeX-LaTeX-sentinel
    (around mg-TeX-LaTeX-sentinel-open-output activate)
  "Open output when there are errors."
  ;; Run `TeX-LaTeX-sentinel' as usual.
  ad-do-it
  ;; Check for the presence of errors.
  (when
      (with-current-buffer TeX-command-buffer
        (plist-get TeX-error-report-switches (intern (TeX-master-file))))
    ;; If there are errors, open the output buffer.
    (TeX-recenter-output-buffer nil)))

;; do not check comments
(setq ispell-list-command "--list")
(setq ispell-extra-args '("--dont-tex-check-comments"))

;; Reftex
(add-hook 'LaTeX-mode-hook 'turn-on-reftex)   ; with AUCTeX LaTeX mode
(add-hook 'latex-mode-hook 'turn-on-reftex)   ; with Emacs latex mode

;; text viewer
(setq TeX-view-program-list
      '(("Acrobat" "Acrobat.exe %o")
        ("Gsview" "gsview32.exe %o")
        ("gv" "gv %o")
        ("qpdfview" "qpdfview --unique %o")
        ("MasterPDF" "masterpdfeditor4 %o")
        ("Evince" "evince %o")
        ("Firefox" "firefox %o")
        ("Yap" "yap.exe %o")))
(cond
 ((eq system-type 'windows-nt)
  (add-hook 'LaTeX-mode-hook
            (lambda ()
              (setq TeX-view-program-selection '((output-pdf "Acrobat")
                                                 (output-dvi "Yap"))))))
 ((eq system-type 'gnu/linux)
  (add-hook 'LaTeX-mode-hook
            (lambda ()
              (setq TeX-view-program-selection '((output-pdf "PDF Tools")
                                                 (output-dvi "gv")))))))
;;---------------------------
;; Markdownmode with mathjax
;;---------------------------
(setq markdown-command
      (concat
       "/usr/local/bin/pandoc"
       " --from=markdown --to=html"
       " --standalone --mathjax --highlight-style=pygments"))
;;------------------------------
;; Chinese fonts
;;------------------------------
(defvar emacs-english-font "Source Code Pro"
  "The font name of English.")

(defvar emacs-cjk-font "方正宋刻本秀楷"
  "The font name for CJK.")

(defvar emacs-font-size-pair '(15 . 18)
  "Default font size pair for (english . chinese)")

(defvar emacs-font-size-pair-list
  '(( 5 .  6) (10 . 12)
    (13 . 16) (15 . 18) (17 . 20)
    (19 . 22) (20 . 24) (21 . 26)
    (24 . 28) (26 . 32) (28 . 34)
    (30 . 36) (34 . 40) (36 . 44))
  "This list is used to store matching (englis . chinese) font-size.")

(defun font-exist-p (fontname)
  "Test if this font is exist or not."
  (if (or (not fontname) (string= fontname ""))
      nil
    (if (not (x-list-fonts fontname)) nil t)))

(defun set-font (english chinese size-pair)
  "Setup emacs English and Chinese font on x window-system."

  (if (font-exist-p english)
      (set-frame-font (format "%s:pixelsize=%d" english (car size-pair)) t))

  (if (font-exist-p chinese)
      (dolist (charset '(kana han symbol cjk-misc bopomofo))
        (set-fontset-font (frame-parameter nil 'font) charset
                          (font-spec :family chinese :size (cdr size-pair))))))

;; Setup font size based on emacs-font-size-pair
(when (display-graphic-p)
  (set-font emacs-english-font emacs-cjk-font emacs-font-size-pair))

(defun emacs-step-font-size (step)
  "Increase/Decrease emacs's font size."
  (let ((scale-steps emacs-font-size-pair-list))
    (if (< step 0) (setq scale-steps (reverse scale-steps)))
    (setq emacs-font-size-pair
          (or (cadr (member emacs-font-size-pair scale-steps))
              emacs-font-size-pair))
    (when emacs-font-size-pair
      (message "emacs font size set to %.1f" (car emacs-font-size-pair))
      (set-font emacs-english-font emacs-cjk-font emacs-font-size-pair))))

(defun restore-emacs-font-size ()
  "Restore emacs's font-size acording emacs-font-size-pair-list."
  (interactive)
  (setq emacs-font-size-pair '(15 . 18))
  (when emacs-font-size-pair
    (message "emacs font size set to %.1f" (car emacs-font-size-pair))
    (set-font emacs-english-font emacs-cjk-font emacs-font-size-pair)))

(defun increase-emacs-font-size ()
  "Decrease emacs's font-size acording emacs-font-size-pair-list."
  (interactive) (emacs-step-font-size 1))

(defun decrease-emacs-font-size ()
  "Increase emacs's font-size acording emacs-font-size-pair-list."
  (interactive) (emacs-step-font-size -1))

(when (display-graphic-p)
  (global-set-key (kbd "C-x C-=") 'increase-emacs-font-size)
  (global-set-key (kbd "C-x C--") 'decrease-emacs-font-size)
  (global-set-key (kbd "C-x C-0") 'restore-emacs-font-size))

;;--------------------------
;; emacsclient kill server
;;--------------------------
(defun client-save-kill-emacs(&optional display)
  " This is a function that can bu used to shutdown save buffers and
shutdown the emacs daemon. It should be called using
emacsclient -e '(client-save-kill-emacs)'.  This function will
check to see if there are any modified buffers or active clients
or frame.  If so an x window will be opened and the user will
be prompted."

  (let (new-frame modified-buffers active-clients-or-frames)

                                        ; Check if there are modified buffers or active clients or frames.
    (setq modified-buffers (modified-buffers-exist))
    (setq active-clients-or-frames ( or (> (length server-clients) 1)
                                        (> (length (frame-list)) 1)
                                        ))

                                        ; Create a new frame if prompts are needed.
    (when (or modified-buffers active-clients-or-frames)
      (when (not (eq window-system 'x))
        (message "Initializing x windows system.")
        (x-initialize-window-system))
      (when (not display) (setq display (getenv "DISPLAY")))
      (message "Opening frame on display: %s" display)
      (select-frame (make-frame-on-display display '((window-system . x)))))

                                        ; Save the current frame.
    (setq new-frame (selected-frame))

                                        ; When displaying the number of clients and frames:
                                        ; subtract 1 from the clients for this client.
                                        ; subtract 2 from the frames this frame (that we just created) and the default frame.
    (when ( or (not active-clients-or-frames)
               (yes-or-no-p (format "There are currently %d clients and %d frames. Exit anyway?" (- (length server-clients) 1) (- (length (frame-list)) 2))))

                                        ; If the user quits during the save dialog then don't exit emacs.
                                        ; Still close the terminal though.
      (let((inhibit-quit t))
                                        ; Save buffers
        (with-local-quit
          (save-some-buffers))

        (if quit-flag
            (setq quit-flag nil)
                                        ; Kill all remaining clients
          (progn
            (dolist (client server-clients)
              (server-delete-client client))
                                        ; Exit emacs
            (kill-emacs)))
        ))

                                        ; If we made a frame then kill it.
    (when (or modified-buffers active-clients-or-frames) (delete-frame new-frame))
    )
  )

(defun modified-buffers-exist()
  "This function will check to see if there are any buffers
that have been modified.  It will return true if there are
and nil otherwise. Buffers that have buffer-offer-save set to
nil are ignored."
  (let (modified-found)
    (dolist (buffer (buffer-list))
      (when (and (buffer-live-p buffer)
                 (buffer-modified-p buffer)
                 (not (buffer-base-buffer buffer))
                 (or
                  (buffer-file-name buffer)
                  (progn
                    (set-buffer buffer)
                    (and buffer-offer-save (> (buffer-size) 0))))
                 )
        (setq modified-found t)
        )
      )
    modified-found
    )
  )
